tests/inputData.json.example

```diff
@@ -34,6 +34,13 @@
 		"NSX Edge Nodes": [
 			"sfo-m01-en01",
 			"sfo-m01-en02"
+		],
+		"NSX Edge Fqdn": [
+			"sfo-m01-en01.sfo.rainpole.io",
+			"sfo-m01-en02.sfo.rainpole.io"
```

Comment:
We can look at it later, because this requires some code change. Like just having hostname and then read domain name from SDDC manager and so on. Instead I think I have a ticket to enhance pester to read everything from SDDC manager if possible than from JSON. We can take it up there if you don't mind.